### PR TITLE
[IMP] util: Announce Release Notes after a migration

### PR DESCRIPTION
**Pull request by arbaes**
The goal is to warn users that have just upgraded and get them curious about the new features.

TaskID: 3950689

### CODE REVIEW

File: src/util/report.py [hunk 1]
```diff
@@ -90,27 +115,40 @@ def announce_migration_report(cr):
...
     if not isinstance(message, basestring):
         message = message.decode("utf-8")
     if message.strip():
         message = message.replace("{", "{{").replace("}", "}}")
         kw = {}
         # If possible, post the migration report message to administrators only.
         admin_channel = get_admin_channel(cr)
-        if admin_channel:
+        if admin_channel and to_admin_only:
             kw["recipient"] = admin_channel
```
Comment (KangOl) [inline src/util/report.py:150]:
```suggestion
        recipient = get_admin_channel(cr) if to_admin_only else None
        if recipient:
            kw["recipient"] = recipient
```

**Comment by KangOl** [comment]
It works as expected:
image

**Review by KangOl — APPROVED** [review]
@robodoo r+

File: src/util/report.py [hunk 2]
```diff
@@ -82,6 +90,25 @@ def add_to_migration_reports(message, category="Other", format="text"):
     migration_reports.setdefault(category, []).append((message, raw))
 
 
+def announce_release_note(cr):
```
[the remark this answers is not in the document]
Comment (arbaes) [reply src/util/report.py:93]: As discussed with APR, it's ok with getting a message after each migration step

**Pull request closed without merging**